DFE-Digital/early-years-foundation-reform
# add gem-c-govspeak class to articles

## Ticket and context

Ticket: [HFEYP-628](https://dfedigital.atlassian.net/browse/HFEYP-628)
## Tech review
Add gem-c-govspeak class which makes images responsive. Also adds other styling which should have been there previously

### Code quality checks

## Product review

### How can someone see it in review app?
1. Click the link to go to [article on review app](https://eyfs-cms-review-pr-532.london.cloudapps.digital/articles/internet-safety-a-test-article-about-internet-safety)
2. Verify that image is now responsive on mobile, as required
![image](https://user-images.githubusercontent.com/13471320/148380108-15210e6f-c7ba-487c-81d9-0f027d74b507.png)
3. Click this [link to go to edit page](https://eyfs-cms-review-pr-532.london.cloudapps.digital/admin/articles/internet-safety-a-test-article-about-internet-safety/edit) and verify that styling is the same
4. Click this [link to go to preview page](https://eyfs-cms-review-pr-532.london.cloudapps.digital/admin/articles/internet-safety-a-test-article-about-internet-safety) and verify that styling is the same
